ui/admin/app/components/form/account/oidc/index.hbs

```diff
@@ -10,25 +10,24 @@
   @showEditToggle={{if @model.isNew false true}}
   as |form|
 >
-  <InfoField @value={{@model.type}} readonly={{true}} as |F|>
+  <InfoField @value={{@model.type}} as |F|>
```

Comment:
Are re removing `readonly={{true}}` because the default is already set to `true`?

Reply:
Correct.